plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/names/FreshNames.kt

```diff
@@ -36,6 +36,16 @@ data object ClassPredicateSubjectName : MangledName {
         get() = "class\$predicate\$subject"
 }
 
+data object GetterFunctionSubjectName : MangledName {
```

Comment:
nit: Perhaps this can be an object of `SpecialName`?

Reply:
I am not sure about it, doing so will require to instantiate `SpecialName` with the same string in different parts of the code (`TypeEmbedding` and twice in `SpecialFunctions`)

Reply:
I agree that's not a good solution.  I was thinking `data object GetterFunctionSubjectName : SpecialName("...")` but actually `SpecialName` is already a data class...

Perhaps `SpecialNameBase` should be an abstract class that provides the actual logic that `SpecialName` now provides, and all the single-use names we want (like this one) should be `data object`s deriving from that?  That way we get to use a single implementation instead of having to come up with a mangling scheme each time.